### PR TITLE
Update DG and own md

### PR DESCRIPTION
**Pull request by jundatan**
DG updates:
- Add user cases such as Exit and clear
- Beautify DG

Own md:
- Updated contributions and respective informations.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -373,31 +377,59 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-*{More to be added}*
+> **Use case: UC05 - Clear all clients**
+
+**MSS**
+
+1. User requests to clear the list of clients.
+2. FitBook clears the list and database of clients.
+
+   Use case ends.
+
```
Comment (dfordarius) [inline docs/DeveloperGuide.md:411]:
```suggestion
> **Use case: UC05 - Clear all clients**

    **MSS**

1. User requests to clear the list of clients.
2. FitBook clears the list and database of clients.

   Use case ends.
   
    **Extensions**

    1a. The list is empty in the database 
    
        1a1. FitBook displays list is currently empty message 
```

**Review by dfordarius — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jundatan) [reply docs/DeveloperGuide.md:411]: 1a1. The FitBook dont display that the list is currently empty. It just shows that the list is cleared. Unless you want to add that in.

Comment (dfordarius) [reply docs/DeveloperGuide.md]: mine is just an example, its up to u to change to whatever u deem fit 😄

Comment (jundatan) [reply docs/DeveloperGuide.md]: Alright. Updated the following. 😺

**Review by dfordarius — APPROVED** [review]

**Comment by dfordarius** [comment]
LGTM

**Pull request merged**